src/implementations/vanilla/components/select/select.js

```diff
@@ -923,8 +943,11 @@ export class Select {
         break;
 
       case 'Tab':
-        e.preventDefault();
-        this.input.focus();
+        if (e.shiftKey) {
+          this.closeButton.previousSibling.focus();
```

Comment:
Also in this case we cannot assume that a previousSibling is going to be present. The same we do it also with ArrowLeft.